pulp_2to3_migration/app/migrations/0012_auto_20200513_1311.py

```diff
@@ -0,0 +1,170 @@
+# Generated by Django 2.2.12 on 2020-05-13 13:11
```

Comment:
Why is `django-admin makemigrations` adding this file (even on `master` before I changed anything)?
Is it because I do not have the `pulp_rpm` plugin installed in my box?
Weird.

Reply:
Yes, I believe it's because you don't have pulp_rpm.
If pulp_rpm is not installed, entry_point for it is ignored, thus django can't read models of that plugin and tries to remove them from the db.

Reply:
I will just leave this thread as a reminder to clean up/regenerate the migrations at some point before I convert this to "ready for review".

Reply:
Finally built a pulplift box with all relevant plugins and regenerated the migrations.
So I can resolve this thread.